scarb/src/bin/scarb/commands/new.rs

```diff
@@ -18,7 +19,12 @@ pub fn run(args: NewArgs, config: &Config) -> Result<()> {
             } else {
                 VersionControl::Git
             },
-            snforge: args.init.snforge,
+            snforge: matches!(
+                Ok(args.init.test_runner)
+                    .transpose()
+                    .unwrap_or_else(ask_for_test_runner)?,
+                TestRunner::StarknetFoundry
+            ),
```

Comment:
duplicate change, please deduplicate this

Reply:
We can't use `TestRunner` as an argument to `ops::InitOptions`, as it is not available there. 

Is something like this okay?

```rust
matches!(
    get_or_ask_for_test_runner(args.init.test_runner)?,
    TestRunner::StarknetFoundry
)
```

Reply:
yep